test/feedback/feedback.e2e.spec.js

```diff
@@ -0,0 +1,36 @@
+const mock = require('../e2e/mock-helpers');
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts');
+
+const pause = 500;
+const selectors = {
+  root: '#feedback-root',
+  revealFormButton: '.feedback-button',
+  form: '.feedback-form',
+  formDescription: '.feedback-form textarea',
+  formShouldSendResponse: '#should-send-response',
+  formEmail: '.feedback-form input[name=email]',
+  formSubmit: '.feedback-form button[type=submit]',
+  formSubmitted: '#feedback-submitted'
+};
+
+function begin(browser) {
+  browser.url(E2eHelpers.baseUrl)
+    .waitForElementVisible(selectors.root, Timeouts.slow)
+    .axeCheck('document');
+
+  browser.click(selectors.revealFormButton).pause(pause);
+  browser.setValue(selectors.formDescription, 'This is my feedback').pause(pause);
+  browser.click(selectors.formShouldSendResponse).pause(pause);
+  browser.setValue(selectors.formEmail, 'test@adhocteam.us').pause(pause);
+  browser.click(selectors.formSubmit).pause(pause);
```

Comment:
What are these pauses for?

Reply:
It's to make sure each command is finished before starting the next command, and so that I don't need to use callbacks. Are these not necessary? I may not have tested otherwise.

Reply:
I think the only time you'd need to pause is when you're waiting for an animation to finish. Even then, I think there's a way @jbalboni had to bypass the animation for tests, but I'd have to dig it back up since I don't remember how it worked exactly.

When we're waiting for things, we usually prefer [`.waitForElementVisible()`](http://nightwatchjs.org/api#waitForElementVisible), but that doesn't seem like it'd be useful here.

If you try it without the pauses and it breaks, I'd be really interested to find out why! Seems like it should be fine without pausing.